libvast/src/system/application.cpp

```diff
@@ -343,6 +343,7 @@ auto make_command_factory() {
     {"import test", import_command},
     {"import zeek", import_command},
     {"import zeek-json", import_command},
+    {"import arrow", import_command},
```

Comment:
This is lacking the remote dual to import, namely `spawn source arrow`. This has to be added both in this file and in `node.cpp`.